Gemfile

```diff
@@ -1,6 +1,19 @@
+# frozen_string_literal: true
 source 'https://rubygems.org'
 
 group :jekyll_plugins do
   gem 'github-pages'
-  gem 'jekyll-theme-primer'
+  gem 'jekyll-remote-theme'
+  gem 'webrick'
+  gem 'jekyll-seo-tag'
+  gem 'jekyll-remote-theme'
+  gem 'jekyll-github-metadata'
```

Comment:
Can you clarify which of these are truly new dependencies (and not just transitive ones that we previously got through `jekyll-theme-primer`? Are all the new ones strictly required by your theme?

Reply:
Resolved in [75483e2](https://github.com/tekknolagi/isdt/pull/22/commits/75483e20c82fa2c80b5d7ffbfdfc2f49729d3f9a).